.github/workflows/ci.yaml

```diff
@@ -90,14 +90,14 @@ jobs:
           retention-days: 1
 
   test-binary:
-    # Currently macos-latest is macos-14-arm64, and thus binary build for macos12-arm64 can be tested on it.
+    # Currently macos-latest is macos-15-arm64, and thus binary build for macos14-arm64 can be tested on it.
```

Comment:
Now macos-latest is macos-14: https://github.com/actions/runner-images

Reply:
corrected.